.github/workflows/run-tests.yml

```diff
@@ -61,7 +64,7 @@ jobs:
       - name: Setup
         uses: actions/setup-node@9ced9a43a244f3ac94f13bfd896db8c8f30da67a
         with:
-          node-version: 12.22.10
+          node-version: "${{ steps.parse-node-version.outputs.NVMRC }}"
```

Comment:
Does this share between jobs?

Reply:
🤔 very good question, I'm guessing yes as this worked agains this PR but it would be helpful to prove it!

Reply:
No I've verified that subsequent jobs are just defaulting to whatever behaviour the node runner defaults to. 

This should be updated along with https://github.com/alphagov/pay-products-ui